src/main/java/com/developerphil/adbidea/adb/command/GrantPermissionsCommand.java

```diff
@@ -0,0 +1,74 @@
+package com.developerphil.adbidea.adb.command;
+
+import com.android.ddmlib.IDevice;
+import com.developerphil.adbidea.adb.command.receiver.GenericReceiver;
+import com.intellij.openapi.project.Project;
+import org.jetbrains.android.facet.AndroidFacet;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import static com.developerphil.adbidea.adb.AdbUtil.isAppInstalled;
+import static com.developerphil.adbidea.ui.NotificationHelper.error;
+import static com.developerphil.adbidea.ui.NotificationHelper.info;
+
+/**
+ * Created by Stephan Hagios on 28.07.17.
+ */
```

Comment:
I'd rather not have class headers with creators/dates in the code, they tend to get stale real quick.
Git does a better job a this.
Can you remove them in all files in this PR?